packages/lodestar-cli/src/cmds/account/cmds/wallet/create.ts

```diff
@@ -18,6 +19,15 @@ interface IWalletCreateArgs {
   mnemonicOutputPath?: string;
 }
 
+function validatePassphraseFile(passphraseFile: string): void {
+  if (path.parse(passphraseFile).ext !== ".pass") {
+    throw new YargsError("passphraseFile must end with .pass, make sure to not provide the actual password");
+  }
+  if (!fs.existsSync(passphraseFile)) {
```

Comment:
whats the logic for generating random passphrase file if there isn't any. Ain't this gonna fail with wrong password next time you use it?

Reply:
I didn't write this.  I just moved it because codecov was complaining bout the function where that logic used to be having too many lines.  @dapplion ?

Reply:
You should pull out the second `if` here into a `maybeCreatePassphraseFile`. Also call this function `validatePassphraseFilename`.

So then below, the flow will look like:
```
validatePassphraseFilename(passphraseFile);
maybeCreatePassphraseFile(passphraseFile);
```

Reply:
bump

Reply:
gotchu @wemeetagain

Reply:
@wemeetagain Do you think it's a good idea to create these very small utility functions that are not re-used just to please CodeClimate errors? In my personal opinion, it reduces code readability.

Reply:
Not really. I usually just CodeClimate to help identify _potential_ issues, not enforce the rules rigorously. Would be in favor of reverting this specific change.

Reply:
@3xtr4t3rr3str14l Could you revert these two small functions? Feel free to ignore CodeClimate in cases like these

Reply:
ok cool, will do.  this is the first time i've ran into code climate issues on this repo, so i wasn't sure if it would prevent a merge or not.  will push that reversion here in a sec

Reply:
pushed